### DISCUSSION TITLE
I cant access anything on my account before verifying 2fa, but im not receiving sms.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#123332] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

so a couple days ago i used my other account and noticed a pop up telling me to verify 2fa, i always skipped it, now its saying its mandatory, i tried verifying and i only had the option to verify with sms (keep in mind that i have recovery codes, github mobile, and passkeys) but i was like whatever and decided to just verify with 2fa, the first thing i noticed was that i had a weird number ive never seen linked in my security page, and github was acting like it just wasnt there, +44 7488855300. i couldnt remove it, and when i tried adding in my current phone number (or getting the sms verification thing) it gives me this message
"We tried sending an SMS to your configured number, but we are not authorized to send SMS messages to this recipient. Please contact support if you continue to have problems.", all github pages are impossible to access, including the main page, and except for settings, which allowed me to try and add more or remove security methods, which im able to do, but i cant add or remove phone numbers.
this issue is incredibly frustrating and doesnt make any sense to me.

please help me with this
![Screenshot 2024-05-11 001800](https://github.com/community/community/assets/169284146/8748e2a8-8f10-4b45-b01f-e1d2ea04426c)
![Screenshot 2024-05-11 001815](https://github.com/community/community/assets/169284146/36efcfbd-d5c0-4ea9-add3-827ee7ee00ce)

**Comment by ebndev** [comment]
Hi @yousif518111,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
